### PR TITLE
Fix #180486 www.nowfoods.com

### LINKED ISSUE #180486: www.nowfoods.com
### Issue URL (Ads)
 [https://www.nowfoods.com/products](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.nowfoods.com%2Fproducts)

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Android v4.4.1
System version: | Android 14
Browser: | Chrome
License type: | paid
AdGuard mode: | VPN
HTTPS filtering: | enabled
Stealth mode options: | Block Push API, <br>Block Location API, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties, <br>Hide your User-Agent, <br>Hide your IP address (208.54.104.184), <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (180), <br>Self-destructing first-party cookies (4320), <br>Disable cache for third-party requests, <br>Block trackers
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>filters: `https://filters.adtidy.org/android/filters/15_optimized.txt`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Peter Lowe's Blocklist<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>Adblock Warning Removal List, <br/>Fanboy's Annoyances, <br/>Dandelion Sprout's Annoyances List<br/><br/><b>Security:</b><br/>Online Malicious URL Blocklist, <br/>Scam Blocklist by DurableNapkin, <br/>uBlock Origin – Badware risks, <br/>Phishing URL Blocklist<br/><br/><b>Other:</b><br/>Filter unblocking search ads and self-promotion
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js<br>https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.meta.js

### PR DESCRIPTION
**Pull request by sfionov**
One for the cookies, one for the promo alert

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -15421,6 +15421,8 @@ movie-censorship.com##td[width^="160"][align="center"] > div[style*="border: 1px
 tubesex.me/eureka
 enworld.org##ul > li[class="postbitlegacy postbitim postcontainer"]
 pornhd.com##ul.thumbs + div.listing-zone.mobile-zone
+nowfoods.com##.block-sd-cookie-consent-block
```
Comment (jellizaveta) [inline BaseFilter/sections/specific.txt:15424]: Cookies should be in `AnnoyancesFilter/cookies/cookies_specific.txt
Promo popup in `AnnoyancesFilter/popups/popups_specific.txt`

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (jellizaveta) [reply BaseFilter/sections/specific.txt:15424]: Could you also place the rules at the beginning of the lists

**Comment by jellizaveta** [comment]
share button can be blocked
<details><summary>Screenshot:</summary>

Screenshot 2024-06-05 at 2 19 28 PM

</details><br/>

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (sfionov) [reply BaseFilter/sections/specific.txt]: Fixed

**Comment by sfionov** [comment]
@jellizaveta What kind of filter is this? Annoyance -> Widgets? I do not see section for share buttons there :harold:

File: AnnoyancesFilter/Widgets/sections/widgets.txt [hunk 2]
```diff
@@ -1828,6 +1828,8 @@ zebratechnologies.force.com##.zbrbot-container
 !
 !   SECTION: Uncategorized
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/180486 - share button
+nowfoods.com###block-sdsocialshareblock
```
Comment (AdamWr) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:1832]: I think it should be added to `Social Media filter`.
Could you please move it somewhere here (without a link to issue) - https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt#L77

**Review by AdamWr — CHANGES_REQUESTED** [review]

**Review by AdamWr — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**